persistence/local/module.go

```diff
@@ -0,0 +1,80 @@
+package local
```

Comment:
Thank you for the clarification. Will stick to opening new issues/PRs on future PRs to avoid introducing any new changes (already on that trajectory on this PR with multiple issues opened to track out-of-scope changes)